ministryofjustice/correspondence_tool_staff
# CT-3116 Add extra fields to the closed-report for FOI

## Description

This PR is to make sure the following fields to have correct value and will be shown on closed-reports for FOI (business days)
- num_days_taken
- num_days_late
- num_days_taken_after_extention
and those fields will map to the following keys in warehouse table 
- number_of_days_taken
- number_of_days_late
- number_of_days_taken_after_extension

And because there have been case being syned into warehouse table in the past,  in order to refresh the values in old cases
2 extra cts command have been added to allow generating the CaseSyncJob manually based on the options below
- id range of cases
- case number

## Self-review checklist


### Screenshots

Before 
<img width="556" alt="Screenshot 2020-09-25 at 12 29 38" src="https://user-images.githubusercontent.com/26111671/94262086-cd855c80-ff2a-11ea-8553-a954e86b928b.png">
After 
<img width="493" alt="Screenshot 2020-09-25 at 12 18 22" src="https://user-images.githubusercontent.com/26111671/94262095-d118e380-ff2a-11ea-9b53-810c630fe734.png">

### Related JIRA tickets

https://dsdmoj.atlassian.net/secure/RapidBoard.jspa?rapidView=25&modal=detail&selectedIssue=CT-3116
### Deployment

please run db migration to add new fields into warehouse table
rails db:migrate
### Manual testing instructions

1 - Login as London user
2.- Click 'Reports' from top menu
3 - Click 'Create report' button
4 - Choose 'General closed cases report'
5 - Put proper time frame
6 - Click 'Create custom report'
7 - Download the report (in zip format, need to uncompress it) and check the data
